setup.py

```diff
@@ -7,15 +7,20 @@
 from shutil import which
 from skbuild import setup
 
-# CMake Options
 CSP_USE_VCPKG = os.environ.get("CSP_USE_VCPKG", "1").lower() in ("1", "on")
+# Allow arg to override default / env
+if "--csp_no_vcpkg" in sys.argv:
```

Comment:
very small nit but should it be `--csp-no-vcpkg` instead of underscores? Also if this is just for windows for setting in a single line doesn't `set CSP_USE_VCPKG=0 && python setup.py ...` work?

Reply:
I tend to use underscores but agreed this should match the setup.py convention
I had to add this because I couldnt get the env var to pass through in the conda build for some reason, after numerous attempts.  At first I thought it was because it was using powershell, but after getting it to use cmd it still didnt seem to work.  feel free to pursue